vms/platformvm/block/builder/helpers_test.go

```diff
@@ -183,6 +189,7 @@ func addSubnet(t *testing.T, env *environment) {
 
 	stateDiff.AddTx(testSubnet1, status.Committed)
 	require.NoError(stateDiff.Apply(env.state))
+	require.NoError(env.state.Commit())
```

Comment:
this commit is needed to properly consume utxos used to fund subnet creation. Without this, a tx creation may end up using consumed utxos (not properly stored on state) and fail.
The reason that didn't happen before is that subnet creation fee used to be set to zero, which masked the issue.